src/vcpkg/build.cpp

```diff
@@ -1096,6 +1095,14 @@ namespace vcpkg
         std::string portfile_cmake_contents;
         std::vector<Path> files;
         std::vector<std::string> hashes;
+        size_t i = 0;
+        for (auto& file : abi_info.pre_build_info->hash_additional_files)
+        {
+            const auto hash =
+                vcpkg::Hash::get_file_hash(fs, file, Hash::Algorithm::Sha256).value_or_exit(VCPKG_LINE_INFO);
```

Comment:
We should explicitly root the file paths somewhere, instead of leaving it up to "??? cwd"

Reply:
Could you define `somewhere` ? I would also like to see that `somewhere` then be applied to a relative `VCPKG_CHAINLOAD_TOOLCHAIN_FILE`

I purposely left relatives path in the undefined behavior land since in the case of `VCPKG_CHAINLOAD_TOOLCHAIN_FILE` that is basically wherever cmake (script or build) gets to be executed while for `VCPKG_HASH_ADDITIONAL_FILES` it is wherever vcpkg itself gets executed. 
As such disallowing relative paths might be a valid option?